index.js

```diff
@@ -197,13 +208,13 @@ async function analyzeFile(cwd, file, options) {
   }
 }
 
-async function analyzeJsFile(content) {
+async function analyzeJsFile(content, config) {
   let translationKeys = new Set();
 
   // parse the JS file
   let ast = BabelParser.parse(content, {
     sourceType: 'module',
-    plugins: ['decorators-legacy', 'dynamicImport', 'classProperties'],
+    plugins: ['decorators-legacy', 'dynamicImport', 'classProperties', ...config.userPlugins],
```

Comment:
💅🏽 niptick: passing `userPlugins` would be enough here. Or do you want to be ready for reading future config options?

```diff
- async function analyzeJsFile(content, config) {
+ async function analyzeJsFile(content, userPlugins) {
  let translationKeys = new Set();

  // parse the JS file
  let ast = BabelParser.parse(content, {
    sourceType: 'module',
    plugins: ['decorators-legacy', 'dynamicImport', 'classProperties'],
-    plugins: ['decorators-legacy', 'dynamicImport', 'classProperties', ...config.userPlugins],
+    plugins: ['decorators-legacy', 'dynamicImport', 'classProperties', ...userPlugins],
```

Reply:
I've updated it  to only pass in userPlugins so i am going to resolve this comment. Thanks for the review!